rekolekcje-webapp/src/app/participants/participant-details/participant-details.component.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ParticipantDetailsComponent } from './participant-details.component';
```

Comment:
Niech wszędzie będzie formatowanie typu import { coś_tam } from './coś_tam'; zaraz podeślę na slacku jak to zrobić w IntelliJ